src/renderer/views/Watch/Watch.js

```diff
@@ -626,7 +626,7 @@ export default Vue.extend({
             this.enableLegacyFormat()
           }
 
-          if (result.storyboards?.type === 'PlayerStoryboardSpec') {
+          if (result.storyboards?.type === 'PlayerStoryboardSpec' && result.storyboards.boards.length >= 3) {
```

Comment:
Did you get errors from this code so you added this check or did you want to make the if statement safer?

Reply:
I got undefined errors. Shorts I was testing do not have any storyboards. So, the next line was passing undefined to `createStoryboardUrls`. So 'yes' to both.

Reply:
Could you provide some of the links of the shorts that were erroring, the ones I tested (Linus Tech Tips YouTube channel) all worked, so I would like to see if they have other storyboards.

Reply:
I already don't understand the `result.storyboards?.type === 'PlayerStoryboardSpec'` here
With the new `result.storyboards.boards.length >= 3` it's even harder to understand, add some code comments?

Reply:
I missed `result.storyboards.boards[2]` below oops
I find this project lacks code comments very much 😬